### PR TITLE
Add episode description and transcript for 340

### PR DESCRIPTION
**Pull request by nellshamrell**

### CODE REVIEW

File: _transcripts/016-twir-340.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+We have several videos in this week’s issue - including “Bringing WebAssembly to the web with WASI - the Web Assembly System Interface” by Lin Clark and “Microsoft’s Safe Systems Programming Languages Effort” by Dean Bryen and Ryan Levick. There are several other great beginning videos on Interation by Tim McNamara and even more videos in the newsletter.
+
+Want to get started contributing to Rust Open Source projects? The Call for Participation section features issues from pjama, mdbx-rs, ruma, time-rs, and http-types. Click on them and take a look, hopefully you’ll be motivated to work through them and get that pull request in!
+
+The RFC - or request for comment - process is at the core of how the Rust community works. There were no RFCs in final comment period  - or fcp - this week, though there are several tracking issues and pull requests to look at. Each of these is labeled “disposition: merge” - which means they are in pre-fcp or fcp and are likely on their way to being merged in. Check them out - it’s a great way to get to know the internals of the Rust programming language.
```
Comment (ericseppanen) [inline _transcripts/016-twir-340.md:28]: I'd suggest emdash ("—") for these.

File: _episodes/016-twir-340.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: "This Week in Rust - Issue 340"
+date: 2020-05-27T14:30:00Z
+file:  https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-26.mp3
+duration: "05:24"
+length: "7599000"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Nell Shamrell-Harrington - lead editor of This Week in Rust - takes you through highlights from this week's issue of TWiR.
```
Comment (jonhoo) [inline _episodes/016-twir-340.md:10]: Probably em-dashes here too —

File: _episodes/016-twir-340.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+- [Compiling Rust binaries for Windows 98 SE and more: a journey](https://seri.tools/blog/compiling-rust-for-legacy-windows/)]
+- [Conway's Game of Life on the NES in Rust](https://gridbugs.org/conways-game-of-life-on-the-nes-in-rust/)
+- [Writing Python inside your Rust code — Part 4](https://blog.m-ou.se/writing-python-inside-rust-4/)
+- [Zero To Production #0: Foreword](https://www.lpalmieri.com/posts/2020-05-24-zero-to-production-0-foreword/)
+- [How to organize your Rust tests](https://blog.logrocket.com/how-to-organize-your-rust-tests/)
+- [Rust Macro Rules in Practice](https://dev.to/sassman/rust-macro-rules-in-practice-40ne)
+- [Bringing WebAssembly outside the web with WASI by Lin Clark](https://www.youtube.com/watch?v=fh9WXPu0hw8)
+- [Microsoft's Safe Systems Programming Languages Effort](https://mybuild.microsoft.com/sessions/61de34c5-b111-4ece-928f-541854875862?source=sessions)
+- [3 Part Video for Beginners to Rust Porgramming on Iteration](https://tim.mcnamara.nz/post/618982870485172224/rust-iteration)
```
Comment (jonhoo) [inline _episodes/016-twir-340.md:40]: This should probably say "programming" :sweat_smile:

File: _episodes/016-twir-340.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: "This Week in Rust - Issue 340"
+date: 2020-05-27T14:30:00Z
+file:  https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-26.mp3
```
Comment (jonhoo) [inline _episodes/016-twir-340.md:4]: I think CI is choking on the extra whitespace between the `:` and the URL here.

File: _episodes/016-twir-340.md (hunk 3 above)
Comment (nellshamrell) [reply _episodes/016-twir-340.md:40]: gah!

File: _episodes/016-twir-340.md (hunk 4 above)
Comment (nellshamrell) [reply _episodes/016-twir-340.md:4]: Ah! I'll get that fixed!

**Comment by jonhoo** [comment]
Wohoo, out it goes :rocket:

**Pull request merged**